### DISCUSSION TITLE
JDBC URL issue

### DISCUSSION THREAD

**Discussion by dargi98** [sijms/go-ora#538] [Q&A] [answered]
I'm connecting to an Oracle that has two nodes. This JDBC URL works: (DESCRIPTION=(ADDRESS_LIST=(LOAD_BALANCE=yes)(FAILOVER=on)(ADDRESS=(PROTOCOL=TCP)(HOST=host1.domain.com)(PORT=xxxx))(ADDRESS=(PROTOCOL=TCP)(HOST=host2.domain.com)(PORT=xxxx)))(CONNECT_DATA=(SERVICE_NAME=ServiceName)))

But this one doesn't: (DESCRIPTION_LIST=(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=host1.domain.com)(PORT=xxxx))(CONNECT_DATA=(SERVICE_NAME=ServiceName)))(DESCRIPTION=(ADDRESS=(PROTOCOL=TCP)(HOST=host2.domain.com)(PORT=xxxx))(CONNECT_DATA=(SERVICE_NAME=ServiceName))))

Any insight on why the second one is not working? The error I get is ORA-12564: TNS connection refused.

@sijms

**Comment by sijms** [comment]
I use regular expression to detect server, port and service from JDBC string so I will update the code to detect and telll you

**Comment by sijms** [comment, marked as answer]
sorry for late you should change xxxx in expression `port=xxxx` to number example `port=1521` and it will work
